acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -60,6 +60,10 @@
       on master, "gem install sqlite3"
     when /^el-6/
       on master, "yum install -y rubygems ruby-sqlite3 rubygem-activerecord"
+    when /^el-7/
```

Comment:
For el7, we already have the installation of sqlite3 work afaik, why are we doing this differently for AIO specific purposes? I thought we discussed this not being necessary anyway since AIO doesn't need this test.